Modules/socketmodule.c

```diff
@@ -1227,6 +1227,14 @@ makesockaddr(SOCKET_T sockfd, struct sockaddr *addr, size_t addrlen, int proto)
        }
 #endif /* AF_NETLINK */
 
+#if defined(AF_VSOCK)
+       case AF_VSOCK:
+       {
+           struct sockaddr_vm *a = (struct sockaddr_vm *) addr;
+           return Py_BuildValue("II", a->svm_cid, a->svm_port);
```

Comment:
This is the line failing in the Travis. https://travis-ci.org/python/cpython/jobs/248412030#L2063